scipy/stats/_distn_infrastructure.py

```diff
@@ -2853,6 +2854,108 @@ def entropy(pk, qk=None, base=None, axis=0):
     return S
 
 
+def differential_entropy(
+    values: np.ndarray,
+    *,
+    window_length: int = 1,
+    base: Optional[float] = None,
+    axis: int = 0,
+) -> np.ndarray:
+    r"""Calculate the differential entropy of a distribution given a sample.
+
+    This routine uses the Vasicek estimator of the differential entropy. Given
+    a random sample :math:`X_1, \ldots X_n` this is defined as:
+
+    .. math::
+        \frac{1}{n} \sum_1^n \log \left[ \frac{n}{2m} (X_{(i+m)} - X_{(i-m)}) \right]
```

Comment:
I think the qualifiers need some expansion since n+m is outside the array length similarly 1-m is also outside hence padding is not necessarily explained here and renders this formula obsolete.

Reply:
This is now explained.